### ISSUE TITLE
Add a 'help/information' link for all quests

### ISSUE THREAD

**Issue by CloCkWeRX** [westnordost/StreetComplete#94]
Similar:
 * #66 
 * #93 
 * #89

ID does a pretty good job of this by adding an "Info" link into the UI:
![image](https://cloud.githubusercontent.com/assets/365751/24641105/0af8bcee-1940-11e7-910c-5255abb8751a.png)

For an individual quest, it could be useful to add a link into the UI pointing at the relevant openstreetmap wiki page.

IE:
```
public class AddRoadName extends OverpassQuestType {
   public String getHelpLink() {
      return "https://wiki.openstreetmap.org/wiki/Key:highway";
   }
   // ...
}

```

Alternatively, pointing to a specific quest set of documentation (similar to what is done with some HOT tasks) would also be useful.

https://github.com/hotosm/osm-tasking-manager2/issues/979 talks about a similar usability need.

This could likely be represented in the UI in a consistent icon/location on each form.

**Comment by westnordost** [comment]
I do not plan to add this because this app does not expose the user to the "raw" OSM tags, so linking a wiki site that talks about these tags is counter-productive. Rather, I want to design the UI in a way that it is self-explanatory. Where it is not unavoidable, I add short explanation texts (i.e. in the "add road name" form).

Especially in the case of the building levels by the way, it would only confuse the user more (because the app is not asking for the values as they are input via the tags).

What I would probably eventually add, is a small info button for each quest (type), that pops up a dialog that explains in short words:
- why is this mapped/asked at all? What's the use of this?
- some words/pictures to clear up any misunderstandings, like a mini-FAQ for just this quest.

If this is about what you meant, I can leave this issue open, but a wiki link I would definitely not add.

**Comment by CloCkWeRX** [comment]
> If this is about what you meant,
Yeah, that's more the direction I was heading - arbitrary link. 

Check out https://www.openstreetmap.org/user/JanBohmMSF/diary/40072 as a 'how to' guide and it has a built in Q&A section as more of where I was going with it. Obviously, that's aimed at tasking manager and is complex, compared to what the design goal is here.

**Comment by CloCkWeRX** [comment]
Sample explainer text to fix #89 #66 
https://github.com/CloCkWeRX/StreetComplete/wiki/Building-Levels (can turn this into a PR if you want)

**Comment by westnordost** [comment]
No, thank you. As said, I will rather make the quests self-explanatory than to add "guides".
